www/templates/default/UNL/VisitorChat/Message/Edit.tpl.php

```diff
@@ -10,6 +10,5 @@
     </fieldset>
     <input type="hidden" name="conversations_id" value='<?php echo $context->conversations_id ?>'/>
     <input type="hidden" name="_class" value='<?php echo get_class($context->getRawObject()); ?>'/>
-
+    <input id='visitorChat_message_submit' class="wdn-button wdn-button-triad" type="submit" value="Submit" name="visitorChat_message_submit" />
```

Comment:
I'm right there with you. I added this input and named it to be consistent with other similar elements, but I would like to get away from IDs as much as possible, everywhere possible. @mfairchild365 and I have discussed rewriting the markup, but realized that it's going to be quite an undertaking—something that's out of the scope of this visual refresh. Hopefully summer?